### DISCUSSION TITLE
PrimeNG migration guide version >12

### DISCUSSION THREAD

**Discussion by miruritira** [primefaces/community#405] [PrimeNG]
Hi,

Migration guide only shows steps up to version 12 of PrimeNG.
[https://github.com/primefaces/primeng/w ... tion-Guide](https://github.com/primefaces/primeng/wiki/Migration-Guide)

The changelog of each version is available
[https://github.com/primefaces/primeng/b ... ANGELOG.md](https://github.com/primefaces/primeng/blob/master/CHANGELOG.md)

but is there any guidelines explaining how to migrate to versions >12?

Thanks,
Mikel.

**Comment by cetincakiroglu** [comment]
Hi,

We're preparing a section about "How to" guides and it will include a migration guide as well. For now, you can just update your project to angular16 and primeng 16, complete the installation section https://primeng.org/installation. Be careful about the breaking changes in the version.
